examples/tomography.py

```diff
@@ -21,7 +21,7 @@
 """
 
 # Imports for common Python 2/3 codebase
-from __future__ import print_function, division
+from __future__ import print_function, division, absolute_import
```

Comment:
It seems this is a general change, I wont comment on more than this one, but an explanation please.